### PR TITLE
MRG: add doc deploy

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by mathurinm** [comment]
So the doc builds @agramfort 
What is left to do to make it deploy only when commiting to master ?
I have added:
```
filters:
  branches:
    only: master
```
but the deploy is still happening at every branch push

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,15 +62,35 @@ jobs:
...
+              if [[ ${CIRCLE_BRANCH} == "circleci_doc" ]]; then
+                set -e
+                mkdir -p ~/.ssh
+                echo -e "Host *\nStrictHostKeyChecking no" > ~/.ssh/config
+                chmod og= ~/.ssh/config
+                cd doc;
+                pip install ghp-import;
+                make install
+              fi
+            filters:
```
Comment (mathurinm) [inline .circleci/config.yml:91]: @agramfort this line doesn't seem to have an effect

**Comment by agramfort** [comment]
Merge?

**Comment by mathurinm** [comment]
Thank you @agramfort

**Comment by agramfort** [comment]
👍

**Pull request merged**